packages/firestore/src/util/log.ts

```diff
@@ -19,37 +19,42 @@
 import { SDK_VERSION } from '../core/version';
 import { AnyJs } from './misc';
 import { PlatformSupport } from '../platform/platform';
+import { Logger, LogLevel as _LogLevel } from '@firebase/logger';
+
+const client = new Logger();
 
 export enum LogLevel {
   DEBUG,
   ERROR,
   SILENT
 }
 
-let logLevel = LogLevel.ERROR;
-
 // Helper methods are needed because variables can't be exported as read/write
 export function getLogLevel(): LogLevel {
-  return logLevel;
+  if (client.logLevel === _LogLevel.DEBUG) {
+    return LogLevel.DEBUG;
+  }
+
+  if (client.logLevel === _LogLevel.SILENT) {
+    return LogLevel.SILENT;
+  }
+
+  return LogLevel.ERROR;
 }
 export function setLogLevel(newLevel: LogLevel): void {
-  logLevel = newLevel;
+  client.logLevel = newLevel;
```

Comment:
Shouldn't this do the inverse LogLevel => _LogLevel mapping as getLogLevel()?

Reply:
Technically we don't have to because the 3 log types that Firestore supports are supported by the Logger package itself. This just works.

Reply:
Ahh, and TypeScript is validating that I guess (if you added a different entry to Firestore's LogLevel, typescript would complain?).

If so, cool though I'd still probably add a comment to that effect: 

```
// Since Firestore's LogLevel enum is a pure subset of Firebase's LogLevel, we can return it directly.
```

Reply:
Done